src/log.ts

```diff
@@ -146,10 +146,22 @@ export namespace message {
   // SCALE
   export const CANNOT_UNION_CUSTOM_DOMAIN_WITH_FIELD_DOMAIN = 'custom domain scale cannot be unioned with default field-based domain';
 
-  export function CANNOT_USE_SCALE_PROPERTY_WITH_NON_COLOR(prop: string) {
+  export function cannotUseScalePropertyWithNonColor(prop: string) {
```

Comment:
Feel free to fix it.   (I didn't write this method ... and don't feel like fix it now.)  I'm still in the middle of sth else.)